host/xtest/regression_1000.c

```diff
@@ -2687,3 +2694,335 @@ static void xtest_tee_test_1036(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1036, xtest_tee_test_1036,
 		 "Test PAuth (Pointer Authentication)");
+
+#ifdef CFG_ATTESTATION_PTA
```

Comment:
I'd rather test if the PTA is available and skip the test if unavailable as we do in other tests.

Reply:
Right, updated.